drivers/hmis_csv_twenty_twenty_two/spec/models/exporter/coc_code_override_spec.rb

```diff
@@ -23,25 +23,50 @@ def override_coc_codes
     end
   end
 
+  def warehouse_fixture
+    @warehouse_fixture ||= PgFixtures.new(
+      directory: 'drivers/hmis_csv_twenty_twenty_two/spec/fixpoints',
+      excluded_tables: default_excluded_tables,
+      model: GrdaWarehouseBase,
+    )
+  end
+
+  def app_fixture
+    @app_fixture ||= PgFixtures.new(
+      directory: 'drivers/hmis_csv_twenty_twenty_two/spec/fixpoints',
+      excluded_tables: ['versions'],
+      model: ApplicationRecord,
+    )
+  end
+
   before(:all) do
+    self.use_transactional_tests = false
     cleanup_test_environment
-    setup_data
-    override_coc_codes
+    if warehouse_fixture.exists? && app_fixture.exists?
+      warehouse_fixture.restore
+      app_fixture.restore
+    else
+      setup_data
+      override_coc_codes
 
-    @exporter = HmisCsvTwentyTwentyTwo::Exporter::Base.new(
-      start_date: 1.week.ago.to_date,
-      end_date: Date.current,
-      projects: [@projects.first.id],
-      period_type: 3,
-      directive: 3,
-      user_id: @user.id,
-    )
-    @exporter.export!(cleanup: false, zip: false, upload: false)
+      @exporter = HmisCsvTwentyTwentyTwo::Exporter::Base.new(
```

Comment:
Should this be outside the if?

Reply:
Only matters if the fixtures are there, but yeah, it probably should be.